containers/hawkular-metrics-docker/pom.xml

```diff
@@ -95,6 +95,27 @@
       </build>
     </profile>
     <profile>
+      <id>docker-push</id>
+      <build>
+        <plugins>
+          <plugin>
+            <groupId>org.jolokia</groupId>
+            <artifactId>docker-maven-plugin</artifactId>
+            <version>${docker-maven-plugin.version}</version>
```

Comment:
You don't need to declare the version if the plugin is declared in the pluginsManagement/plugins section